scripts/release-post.js

```diff
@@ -32,16 +32,21 @@ function sendRequest (opts) {
     const options = Object.assign({
       headers: { 'User-Agent': 'nodejs.org release blog post script' }
     }, opts)
-
-    request(options, (err, res, body) => {
+    fetch(options.url, options).then(resp => {
+      if (resp.status !== 200) {
+        return reject(new Error(`Invalid status code (!= 200) while retrieving ${options.url}: ${resp.status}`))
+      }
+      if (options.json) {
```

Comment:
I would simplify this as follow:

```js
return options.json ? resp.json() : resp.text()
```

The added benefit is that rejections will be handled.